balena/models/device.py

```diff
@@ -852,6 +856,13 @@ def get_status(self, uuid):
         if device['download_progress'] is not None:
             return DeviceStatus.CONFIGURING
 
+        if device['current_services']:
+            for service_name in device['current_services']:
```

Comment:
if you loop through `for service_name, installs in device['current_services'].items()` here, you can drop the separate assignment in line 861.

Reply:
@xginn8, `dict.items()` is a bit different between Python 2.x and 3.x. Since our code should be compatible with both so I think it's better to implement it like that.

Reply:
@nghiant2710 fwiw I had tested this behavior in both py2/3 before recommending the change (it is compatible), but if you prefer this implementation that's fine with me.